tests/internals.rs

```diff
@@ -60,6 +61,10 @@ lazy_static! {
         felt_str!("1192211877881866289306604115402199097887041303917861778777990838480655617515");
     static ref TEST_ERC20_SEQUENCER_BALANCE_KEY: Felt =
         felt_str!("3229073099929281304021185011369329892856197542079132996799046100564060768274");
+    static ref TEST_ERC20_BALANCE_KEY_UNK_1: Felt =
+        felt_str!("1192211877881866289306604115402199097887041303917861778777990838480655617516");
+    static ref TEST_ERC20_BALANCE_KEY_UNK_2: Felt =
+        felt_str!("3229073099929281304021185011369329892856197542079132996799046100564060768275");
```

Comment:
What does `UNK` mean? `unknown`?

Reply:
I meant `unknown`, but replaced it for `1` and `2`.